content/blogposts/2024/announcing-prompt-library.md

```diff
@@ -0,0 +1,149 @@
+---
+title: "Announcing Prompt Library"
+authors:
+  - name: YK Sugi
+    url: https://x.com/ykdojo
+publishDate: 2024-11-25T10:00-01:00
+description: "Introducing Prompt Library: Save, reuse, and share frequently used prompts for Cody"
+tags: [blog]
+slug: 'announcing-prompt-library'
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/towards-infinite-context-for-code/infinite-context-og.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/towards-infinite-context-for-code/infinite-context-og.png
+---
+
+As you work with Sourcegraph’s Cody, you might find yourself repeating the same prompt over and over again. And you might want to share it with your coworkers or friends.
```

Comment:
```suggestion
As you work with [Cody](https://sourcegraph.com/cody), you might find yourself repeating the same prompt over and over again. Maybe you have an expert prompt engineer on your team that always knows just how to prompt to get the right outcome each time. Wouldn't it be great if you could automate common tasks and share them with your team?```

Reply:
If you can think of any other use cases, feel free to include here, this is a hype-focused blog post announcing the feature, so we can get creative.